static/templates/user_profile_modal.hbs

```diff
@@ -2,6 +2,9 @@
     <div class="modal__overlay" tabindex="-1" data-micromodal-close>
         <div class="modal__container new-style" role="dialog" aria-modal="true" aria-labelledby="dialog_title">
             <div class="modal__header">
+                <div id="user-status" class="tippy-zulip-tooltip" data-tippy-content="{{last_seen}}">
```

Comment:
This ID is too generic and thus risks being a duplicate. 

Also `popover_user_presence` is a bit of a weird name; this widget is the "user_profile", so we should name it `user_profile_presence`.

Reply:
Makes sense, thanks for the changes!